tools/type_extract/include/SvType.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <ostream>
+
+#include "slang/ast/types/Type.h"
+
+namespace CppType {
+enum Type { BOOL, U32, U64, SC_BV, STRUCT, ENUM };
+
+std::string toString(const Type cppType);
```

Comment:
Note that the top-level `const` on value parameters does nothing, so it's a little confusing to include it (because it makes me think that maybe you meant it to be a const reference or something).

Reply:
Forgot the `&`